init_cluster.sh

```diff
@@ -28,7 +28,9 @@ export KUBECONFIG=$PWD/kubeconfig.yaml
 
 kubectl create ns staging
 
-for dir in consul traefik vault monitoring bitwarden ghost; do
+helm repo add pennlabs https://pennlabs.github.io/helm-charts/
```

Comment:
pls change to helm.pennlabs.org

Reply:
Oh rip, that's what happens when I copy/paste without thinking